CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change Log
 
 ## Master
+- Added `showEnvVars` to build scripts to disable printing the environment [351](https://github.com/yonaskolb/XcodeGen/pull/351) @keith
```

Comment:
Should this be under an ` #### Added` section?

Reply:
done